Atos20/indecision-maker-server
# Add questions bd

### PR Template:
#### What's this PR do?   

- Moves our questions dataset from the frontend to the backend
- Adds the migration, data, and seed files for the questions
- Adds questions GET endpoint to our express server 
- Adds testing that database functionality works as expected in the endpoint

#### Where should the reviewer start?    

- Start by rolling back migrations and updating to lates

#### How should this be manually tested?  

- Run `npm test` to check that the new testing block is passing
- Once migrated latest, re-seed and spin the server up
- Interact with the new endpoint via postman and check Postico respectively

#### Any background context you want to provide?  

- Was debating between determining the logic on the backend but decided against it

#### What are the relevant tickets? 

closes #23 
closes #24 
closes #25 

#### Screenshots (if appropriate) 

![Screen Shot 2020-11-02 at 7 20 41 PM](https://user-images.githubusercontent.com/65369275/97933348-80be4e00-1d40-11eb-9aa3-1d0e279b6632.png)
